service/frontend/accessControlledHandler.go

```diff
@@ -706,6 +818,11 @@ func (a *AccessControlledWorkflowHandler) StartWorkflowExecution(
 		Permission:   authorization.PermissionWrite,
 		WorkflowType: request.WorkflowType,
 	}
+
+	if isManual(ctx) {
+		attr.RequestBody = request
```

Comment:
For all the requests that can contain PII, add a comment here that the authorizer should use the new function you introduced while logging requests to avoid revealing private information.

Reply:
done